008-infinispan-getting-started/pom.xml

```
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.quarkus.qe</groupId>
+    <artifactId>beefy-scenarios</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>008-infinispan-getting-started</artifactId>
+
+  <properties>
+    <testcontainers.version>1.14.1</testcontainers.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-infinispan-client-deployment</artifactId>
```

```suggestion
      <artifactId>quarkus-infinispan-client</artifactId>
```
-deplyment artifacts are internal ones